wikitongues/wikitongues/data_store/airtable/airtable_item_id_provider.py

```diff
@@ -0,0 +1,18 @@
+class AirtableItemIdProvider:
```

Comment:
I'm not sure I understand what role the provider plays. Is it purely a json formatter class?

Reply:
My thought was that we use a formula in Airtable (similar to Excel) to assign identifiers, so I’d write a Python class to form identifiers according to the formula in order to query existing items. That way if we ever decide to change how our Airtable works, it is easier to change the code accordingly

Reply:
Ok, so if the identifier is something coming from Airtable and not defined here, then that makes more sense of why it's a provider. Sounds good!